sechub-pds/src/main/resources/application-pds_integrationtest.yml

```diff
@@ -6,11 +6,10 @@ server:
   address: 127.0.0.1
 
 # next lines are defaults when running inside IDE. Links to sechub-integration parts
-sechub.pds.config.file: "./../sechub-integrationtest/src/main/resources/pds-config-integrationtest.json"
-sechub.pds.workspace.rootfolder: "./../sechub-integrationtest/build/test-results/pds-runtime"
+pds.config.file: "./../sechub-integrationtest/src/main/resources/pds-config-integrationtest.json"
+pds.workspace.rootfolder: "./../sechub-integrationtest/build/test-results/pds-runtime"
 
-sechub:
- pds:
+pds:
    config:
```

Comment:
Because we had no coding convention for YAML I created #2332
and added it to this PR.

The mentioned files are now following the new coding standard.
Other files are not following up the coding standard, but I created issue #2333 to handle this as well (but not inside the PR)